### ISSUE TITLE
[BUG] Change "Number of Pages" to "Listening Time" for Audiobooks

### ISSUE THREAD

**Issue by RequestPrivacy** [mateusz-bak/openreads-android#345]
Firstly a "hello" from a new user :wave: . A pleasure to discover the app, so already thanks for the development!

**Describe the bug**
If I enter/edit an audiobook the "size" of the book feels more natural to be tracked as a time unit, not the page numbers of the physical version imho. So I expected to enter a "Listening Time" in the edit screen (i.e. the absolute time it takes to complete).

**To Reproduce**
- Click add FAB
- Choose e.g. "Add book manually"
- Select "Audiobook" as type
- "Number of pages" remain as size info from default "Paper book" type

**Expected behavior**
Once switched to "Audiobook" I expected to be able to select a time unit (hh:mm) as the "length" of the book.

**Screenshots**
x

**Additional info(please complete the following information):**
 - Manufacturer and model: x
 - Android version: x
 - App version: 2.1.0
 - App source (F-Droid, Google Play, Github): Github

**Additional context**
If switching to a time unit for audiobooks the statistics page needs to be changed as it's based on "page numbers" only - as far as I can see. I'm not sure what's the most natural way of combining time and page information into one statistic but for the time being it might be the easiest way to allow entering both units for audiobooks, "Page numbers" of the equivalent paper version and "Listening time"  and still base the statistics on the page number only. So one can already add the "Listening Time" for future use.

**Comment by apobrt** [comment]
Not particularly sure it's a bug, but I totally agree with you, the label should change depending on the medium

**Comment by RequestPrivacy** [comment]
Yes, not really a bug. But also not quite a feature request I imho, but the template just let me choose between those two choices.

**Comment by mateusz-bak** [comment]
Make sense to separate pages and hours - that will cause some logic changes for statistics I think. Changing to feature requets

**Comment by monkeywithacupcake** [comment]
I see the referenced [commit that added listening_time as a variable](https://github.com/Gildas-GH/openreads/commit/168c2ac0fdce7a8b07b9eafbe00f2cbb849fcfe0), but I don't see listening_time in the [openreads csv format](https://github.com/Gildas-GH/openreads/blob/master/doc/csv.md) - I think this might just be a documentation oversight, but I haven't dug into the code enough to be sure.
